### PR TITLE
tools: fix fpgainfo phy command for N6010 card

### PR DESCRIPTION
**Pull request by anandaravuri**
- add list of profiles supported in N6010 ethernet
  - fix port profile offset

Signed-off-by: anandaravuri <ananda.ravuri@intel.com>

### CODE REVIEW

File: tools/libboard/board_n6010/board_n6010.c [hunk 1]
```diff
@@ -398,20 +372,28 @@ fpga_result print_phy_info(fpga_token token)
...
+
+		if (port_profile.profile > HSS_PORT_PROFILE_SIZE) {
+			printf("Port%-28d :%s\n", i, "N/A");
+			continue;
 		}
+
+		for (int j = 0; j < HSS_PORT_PROFILE_SIZE; j++) {
+			if (hssi_port_profiles[j].port_index == port_profile.profile) {
+				printf("Port%-28d :%s\n", i, hssi_port_profiles[j].profile);
+			}
```
Comment (tswhison) [inline tools/libboard/board_n6010/board_n6010.c:395]: Do you want to break out of the loop after it's found?

Comment (tswhison) [reply tools/libboard/board_n6010/board_n6010.c:396]: I think you want break.  I think continue will just keep iterating through the inner loop.

Comment (anandaravuri) [reply tools/libboard/board_n6010/board_n6010.c:396]: fixed, added break to exit for loop

File: tools/libboard/board_n6010/board_n6010.c [hunk 2]
```diff
@@ -101,6 +101,52 @@ struct hssi_port_attribute {
...
+	{.port_index = 20, .profile = "10GbE"},
+	{.port_index = 21, .profile = "25GbE"},
+	{.port_index = 22, .profile = "40GCAUI-4"},
+	{.port_index = 23, .profile = "50GAUI-2"},
+	{.port_index = 24, .profile = "50GAUI-1"},
+	{.port_index = 25, .profile = "100GAUI-1"},
+	{.port_index = 26, .profile = "100GAUI-2"},
+	{.port_index = 27, .profile = "100GCAUI-4"},
+	{.port_index = 28, .profile = "200GAUI-2"},
+	{.port_index = 20, .profile = "200GAUI-4"},
```
Comment (tswhison) [inline tools/libboard/board_n6010/board_n6010.c:144]: .port_index = 29

File: tools/libboard/board_n6010/board_n6010.c [hunk 3]
```diff
@@ -101,6 +101,52 @@ struct hssi_port_attribute {
...
 };
 
+typedef struct hssi_port_profile {
+
+	uint32_t port_index;
+	char profile[FPGA_VAR_BUF_LEN];
+
+} hssi_port_profile;
+
+#define HSS_PORT_PROFILE_SIZE 32
```
Comment (tswhison) [inline tools/libboard/board_n6010/board_n6010.c:111]: Should be 33, based on the table size.

Comment (anandaravuri) [reply tools/libboard/board_n6010/board_n6010.c]: fixed , updated to 33

File: tools/libboard/board_n6010/board_n6010.c (hunk 2 above)
Comment (anandaravuri) [reply tools/libboard/board_n6010/board_n6010.c]: changes 20 to correct index 29

**Review by tswhison — APPROVED** [review]

**Review by michelleyho — APPROVED** [review]

**Pull request merged**